packages/components/src/utils/getPropsFrom.js

```diff
@@ -14,7 +14,7 @@ function extractComponentProps(Component, props) {
 	return extractedProps;
 }
 
-function extractNativeProps(props) {
+export function extractNativeProps(props) {
```

Comment:
can we use this opportunity to add some docs on why this function exist and how it affect the shape of props ?
Since its a reusable api i think it should be properly documented so the why to use is obvious.

Reply:
Sorry, I reverted what I did with that.
I revert this file